readme.md

```diff
@@ -26,7 +32,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+I'm not quite sure i understand this question. Is it referring to how we can make a post request to the server? Because we can use AJAX for that. Is it talking about submitting an actual form? Because all we would need to do is select that form and call submit on it - e.g. $('#form').submit();
```

Comment:
When submitting a form via AJAX, there are a few things we need to do:
1. Register an event listener for the form submissions (buttton click).
2. When the `submit` happens, we need to preventDefault() to keep the browser from submitting the form.
3. Next, we need to collect all the data from the form, and package it up into an object to be submitted by ajax. We can use `.val()` to get the value of an input
4. Finally, we use $.ajax() with a post/put verb, to make the ajax request. We also need to include a `data` property which contains the form data.